site/pages/home/newsletter-slice/index.js

```diff
@@ -61,7 +69,7 @@ class NewsLetter extends Component {
     }
   };
 
-  updateUser = data => {
+  updateUser = (data: any) => {
```

Comment:
Can we do better here then `any`? It's pretty much a soft bail out from the type system.